src/server/multi_test.cc

```diff
@@ -381,6 +381,10 @@ TEST_F(MultiTest, Eval) {
   ASSERT_THAT(resp, ArrLen(3));
   const auto& arr = resp.GetVec();
   EXPECT_THAT(arr, ElementsAre("a", "b", "c"));
+
+  Run({"zadd", "z1", "123", "a"});
+  resp = Run({"eval", "return redis.call('ZSCORE', KEYS[1], 'a') .. '-is-int'", "1", "z1"});
```

Comment:
I still need to understand. What did it return before?

Reply:
Specifically, maybe you can show the difference with `tonumber` command? I did not understand from your comment above where DF fails

Reply:
123.0-is-int